### PR TITLE
feat(cli): added additional rebuild info to io-engine-client

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

**Comment by dsavitskiy** [comment]
bors try

File: io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs [hunk 1]
```diff
@@ -495,22 +507,28 @@ async fn stats(
...
+                    response.blocks_total.to_string(),
+                    response.blocks_recovered.to_string(),
+                    response.blocks_transferred.to_string(),
+                    response.blocks_remaining.to_string(),
+                    response.progress.to_string(),
+                    response.blocks_per_task.to_string(),
+                    response.block_size.to_string(),
+                    response.is_partial.to_string(),
+                    response.tasks_total.to_string(),
+                    response.tasks_active.to_string(),
```
Comment (tiagolobocastro) [inline io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs:531]: Guess we don't need the map to to_string below anymore?

File: io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs [hunk 2]
```diff
@@ -542,7 +562,7 @@ async fn progress(
 
     let response = ctx
         .client
-        .get_rebuild_progress(rpc::RebuildProgressRequest {
+        .get_rebuild_progress(v0::RebuildProgressRequest {
```
Comment (tiagolobocastro) [inline io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs]: why are we using v0 in v1?

**Review by mtzaurus — APPROVED** [review]

**Review by tiagolobocastro — APPROVED** [review]

File: io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs (hunk 1 above)
Comment (dsavitskiy) [reply io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs:531]: yes, indeed

File: io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs (hunk 2 above)
Comment (dsavitskiy) [reply io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs]: v1 has no "get_rebuild_progress" API.
Ok, I will change to use rebuild_stats().

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Pull request merged**